moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -257,10 +391,11 @@ const NewProjectMap = ({
               data={{
                 ...featureCollection,
                 features: [
-                  ...featureCollection.features.filter(
-                    feature => feature.properties.sourceLayer === sourceName
-                  ),
-                ],
+                  ...featureCollection.features,
+                  ...(otherProjectFeaturesCollection?.features ?? []),
+                ].filter(
+                  feature => feature.properties.sourceLayer === sourceName
+                ),
```

Comment:
Notice line 395, this line is basically copying all the features of the project into a single edit session. The data is initially hidden, but this allows us to show or hide the entire project extent whenever we edit a single component.